msteams-platform/bots/bot-features.md

```diff
@@ -47,6 +48,105 @@ The [Bot Framework](https://dev.botframework.com/) is a rich SDK used to create
 
 Webhooks and connectors connect your bot to your web services. Using webhooks and connectors, you can create a bot for basic interaction, such as creating a workflow or other simple commands. They are available only in the team where you create them and are intended for simple processes specific to your company's workflow. For more information, see [what are webhooks and connectors](~/webhooks-and-connectors/what-are-webhooks-and-connectors.md).
 
+## Azure bot service
+
+The Azure bot service, along with the Bot Framework, provides tools to build, test, deploy, and manage intelligent bots, all in one place. You can also create your bot in Azure bot service.
+
+> [!IMPORTANT]
+> Bot applications within Microsoft Teams are available in GCC-High through [Azure bot Service](/azure/bot-service/channel-connect-teams).
+
+> [!NOTE]
+> * Bots in GCCH only support  up to manifest version v1.10.
+> * Image URL's in Adaptive Cards are not supported in GCCH environment. You can replace an image URL with Base64 encoded DataUri.
+> * Bot channel registration in Azure Government will provision web app bot, app service (app service plan), and application insights also but it doesn't support to provision the azure bot service only (no app service).
+>   <details>
+>   <summary><b>If you want to do bot registration only</b></summary>
+>
+>   * Go to the resource group and manually delete the unused resources. Such as the app service, app service plan (if you created during bot registration), and the application insights (if you choose to enable it during bot registration).
```

Comment:
Brenda, can we get some guidance on this please.

Reply:
It depends on the usage and also the options in the creation flow. e.g., User did not want the app service (Bot app) to be provisioned by ABS. Then the unused resource will be the app service (bot app). Meanwhile in ABS creation, user could choose to create the app service plan or select existing app service plan for defining the compute resources for a web app (Bot app) to run. One or more apps can be configured to run on the same computing resources (in the same App Service plan).  So if user choose to create new app service plan which is the compute resource for bot app and they did not want bot app, the app service plan is unused resource as well. But if user choose the existing app service plan which many apps will use the same one, then this existing app service plan is not in the scope. Same for the Application Insights resource, if user choose to enable it but does not use it for saving conversation logs, this could be removed as well.